### DISCUSSION TITLE
Web Project - Duvida

### DISCUSSION THREAD

**Discussion by EriksonLoures** [V1ctorW1ll1an/WebProject#24] [Q&A] [answered]
Como esta o andamento do projeto?

**Comment by v1ctorw1ll1an** [comment, marked as answer]
@EriksonLoures Estou aguardando o @casnext  para verificar os requisitos e dar continuidade.

**Reply by casnext** [reply]
Daremos continuidade agora, iniciando com todas telas de cadastro

**Comment by v1ctorw1ll1an** [comment]
@casnext precisamos ver como vai ficar os requisitos do cliente

**Reply by casnext** [reply]
@V1ctorW1ll1an podemos iniciar com a tela de cadastro de cliente?

**Reply by v1ctorw1ll1an** [reply]
@casnext Podemos

**Reply by v1ctorw1ll1an** [reply]
Mas voce ja fez o login comunicar com a api?

**Reply by casnext** [reply]
Não somente, criei o front-end, a comunicação tera incio amanha.
